katportalclient/client.py

```diff
@@ -1105,6 +1112,44 @@ def schedule_block_detail(self, id_code):
                 "Invalid schedule block ID: " + id_code)
         raise tornado.gen.Return(schedule_block)
 
+    #TODO: Might need to add a usage script in ../examples
+    @tornado.gen.coroutine
+    def sb_ids_by_capture_block(self, capture_block_id):
+        """Return list of approved observation schedule blocks.
+
+        The schedule blocks have already approved. For detail about
+        a schedule block, use :meth:`.schedule_block_detail`.
+
+        .. note::
+
+            The websocket is not used for this request - it does not need
+            to be connected.
+
+        Parameters
+        ----------
+        capture_block_id: str
+            Capture block identifier. For example: ``1556067480``.
+
+        Returns
+        -------
+        list:
+            List of scheduled block ID strings.  Ordered according to
+            priority of the schedule blocks (first has hightest priority).
```

Comment:
Also they are not ordered by priority, so remove (that comes from the copied docstring)